internal/venice-test-common/src/integrationtest/java/com/linkedin/venice/endToEnd/MetaSystemStoreTest.java

```diff
@@ -147,6 +147,15 @@ public void bootstrapMetaSystemStore() throws ExecutionException, InterruptedExc
         10,
         TimeUnit.SECONDS);
     String metaSystemStoreName = VeniceSystemStoreType.META_STORE.getSystemStoreName(regularVeniceStoreName);
+
+    // Check meta system store property
+    Store metaSystemStore =
+        venice.getLeaderVeniceController().getVeniceAdmin().getStore(venice.getClusterName(), metaSystemStoreName);
+    assertNotNull(metaSystemStore, "Meta System Store shouldn't be null");
+    assertTrue(
+        metaSystemStore.getLatestVersionPromoteToCurrentTimestamp() > 0,
```

Comment:
Not sure if `>0` is enough. In prod we saw a non-0 value even without this piece of code

Reply:
There is a repair at leadership handover of Controller, and this doesn't kick in in the integration test, and in the test, it is always `-1` without the fix.

If you think it is not good enough, we could do another meta system store push, to see whether this field changes or not because of a new push.

Reply:
Yes. Let's trigger a new push and verify that the value changes

I don't understand why we rely on a repair routine. Shouldn't we be fixing the problem at the source? (If we added it to fix some prior issue, and that issue is now fixed, we should look at retiring the routine)